web/etc/proxy/package-lock.json

```diff
@@ -0,0 +1,1235 @@
+{
+    "name": "proxy",
+    "version": "1.0.0",
+    "lockfileVersion": 3,
+    "requires": true,
+    "packages": {
+        "": {
+            "name": "proxy",
+            "version": "1.0.0",
+            "license": "ISC",
```

Comment:
Please check also regarding this license value when everything has been compiled